routes/api.php

```diff
@@ -11,5 +11,5 @@
 
 Route::get('/files', 'FilesController@index');
 Route::post('/files', 'FilesController@store');
-Route::get('/files/{fileKey}', 'FilesController@download')->where('fileKey', '.*');
+Route::get('/files/{fileKey}', 'FilesController@file')->where('fileKey', '.*');
```

Comment:
I don't think we need to change the method name, As _inline_ & _attachment_ are actually downloading the file to the browser.